lib/flay.rb

```diff
@@ -63,7 +63,7 @@ def self.parse_options args = ARGV
 
     OptionParser.new do |opts|
       opts.banner  = "flay [options] files_or_dirs"
-      opts.version = Flay::VERSION
+      opts.version = VERSION
```

Comment:
Not sure I agree with that one... I guess it depends on what you're trying to achieve w/ the subclass?